docs/tutorials/ns-record.md

```diff
@@ -1,6 +1,6 @@
 # Creating NS record with CRD source
 
-You can create NS records with the help of [CRD source](/docs/contributing/crd-source.md)
+You can create NS records with the help of [CRD source](../contributing/crd-source.md)
```

Comment:
this is the best style of markdown link!

Reply:
With this link, it works on both github and mkdocs.